CodeStarterCamp_Week1/main.swift

```diff
@@ -8,5 +8,31 @@
 
 import Foundation
 
-print("Hello, World!")
+func drawIceCream(height: Int, width: Int) {
+    let iceCreamString = String(repeating: "*", count: width)
+    for _ in 0..<height {
+        print(iceCreamString)
+    }
+}
+
+func drawStick(height: Int, width: Int) {
+    let space = (width - 3) / 2
+    let spaceString = String(repeating: " ", count: space)
+    var stickString = ""
```

Comment:
```suggestion
    let stickString: String
```
이런식으로도 작성해 볼 수 있을 것 같습니다!! (절대 틀린것은 아닙니다 🙅)

Reply:
타입 선언만 하고 나중에 할당하면 `let`을 사용할 수도 있군요!! 새로운 방법 알아갑니다😆